src/Test/BlockServiceTestCase.php

```diff
@@ -99,3 +104,22 @@ protected function assertSettings(array $expected, BlockContextInterface $blockC
         $this->assertSame($completeExpectedOptions, $blockSettings);
     }
 }
+
+// NEXT_MAJOR: Remove this hack when dropping support for PHPUnit 7
+if (interface_exists('PHPUnit_Framework_MockObject_MockObject')) {
```

Comment:
Why not `version_compare(\PHPUnit\Runner\Version::id(), '8.0', '<')`?

Reply:
Fixed